mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -99,6 +99,7 @@ CLASS ahl net/minecraft/entity/Entity
 	METHOD a setSize (FF)V
 		ARG 1 width
 		ARG 2 height
+	METHOD a doRelativeMovement (FFFF)V
```

Comment:
`moveRelative`?

Reply:
ehh, that could cause trouble.

Reply:
the `do` prefix is bad and `moveRelative` would fit in along with our pre-existing `move.` Maybe you should change `move` to `doMovement` then but I feel this needs discussing properly.

Reply:
that's fair. This should def be talked over a bit more

Reply:
`moveRelative` seems ok and goes with the current `move`.

`doMovement` is just a verbose way of saying `move`